CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# Unreleased features
```

Comment:
What's the plan with this section? Is it a running list of what's on `main`?

Reply:
- we now have an mdbook! =)
- #127

Reply:
> Is it a running list of what's on main?

Yeah, that's what I had in mind there! But I think it's also fine to just add a new section once we make a release!